bug_crowd/test.py

```diff
@@ -43,6 +44,7 @@ def setUp(self):
         self.api_token = 'api-token-%s' % uuid.uuid4()
         self.client = BugcrowdClient(self.api_token)
         self._bounty = get_example_bounty()
+        self._comments = get_example_comments()
```

Comment:
Assuming that the result of calling `get_example_comments` is always the same we need not create `self._comments` here.